babel.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
           node: "current"
         }
       }
-    ]
+    ],
+    "@babel/preset-typescript"
```

Comment:
This will transpile our TS! We didn't need this with Parcel, but we need it now to support Jest.